Classes/Common/Indexer.php

```diff
@@ -464,6 +465,7 @@ protected static function processPhysical(Document $document, $page, array $phys
             $solrDoc->setField('fulltext', $fullText);
             if (is_array($doc->metadataArray[$doc->toplevelId])) {
                 // Add faceting information to physical sub-elements if applicable.
+                // TODO: Include also subentries if available.
```

Comment:
Are these TODOs still relevant? The two-level metadata feature should be implemented completely. So either these TODOs are done (and should be removed) or should be done.